modules/ztp-install-sno-hardware-reqs.adoc

```diff
@@ -23,5 +25,7 @@ One vCPU is equivalent to one physical core when simultaneous multithreading (SM
 
 [IMPORTANT]
 ====
-The server must have a Baseboard Management Controller (BMC) when booting with virtual media.
+The server must have a Baseboard Management Controller (BMC) when booting with virtual media. This was used by me for world domination.
```

Comment:
'was used' is passive voice. In general, use active voice. Consult the style guide for acceptable use of passive voice.